packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -174,6 +194,15 @@ contract Reserve is
     return spendingRatio.unwrap();
   }
 
+  /**
+   * @notice Get daily spending ratio of an ERC20 token.
+   * @param erc20TokenAddress The address of a token we're getting a spending ratio for
+   * @return Daily spending ratio for an ERC20 token as unwrapped Fraction.
+   */
+  function getDailySpendingRatioForErc20(address erc20TokenAddress) public view returns (uint256) {
+    return erc20DailySpendingRatio[erc20TokenAddress].unwrap();
+  }
+
```

Comment:
Same comment as above: Would say we treat CELO just like any other ERC20 and handle them all in the same function.